backend/workflow_manager/endpoint/destination.py

```diff
@@ -433,3 +473,48 @@ def get_json_schema_for_api(cls) -> dict[str, Any]:
             os.path.dirname(__file__), "static", "dest", "api.json"
         )
         return cls.get_json_schema(file_path=schema_path)
+
+    def _push_to_queue(
+        self,
+        file_name: str,
+        workflow: Workflow,
+        result: Optional[str] = None,
+        input_file_path: Optional[str] = None,
+    ) -> None:
+        """Handle the Manual Review QUEUE result.
+
+        This method is responsible for pushing the input file and result to
+        review queue.
+        Args:
+            file_name (str): The name of the file.
+            result (Optional[str], optional): The result of the API call.
+                Defaults to None.
+
+        Returns:
+            None
+        """
+        if not result:
+            return
+        connector: ConnectorInstance = self.source_endpoint.connector_instance
+        connector_settings: dict[str, Any] = connector.connector_metadata
+
+        source_fs = self.get_fsspec(
+            settings=connector_settings, connector_id=connector.connector_id
+        )
+        with source_fs.open(input_file_path, "rb") as remote_file:
+            file_content = remote_file.read()
+            # Convert file content to a base64 encoded string
+            file_content_base64 = base64.b64encode(file_content).decode("utf-8")
+            q_name = f"review_queue_{self.organization_id}_{workflow.workflow_name}"
+            queue_result = {
+                "file": file_name,
+                "status": QueueResultStatus.SUCCESS,
+                "result": result,
+                "workflow_id": str(self.workflow_id),
+                "file_content": file_content_base64,
+            }
```

Comment:
NIT: Make it a dataclass rather than a loose dict

Reply:
done